game/views.py

```diff
@@ -1,10 +1,125 @@
 from django.shortcuts import render
-from django.views.generic import TemplateView
+from django.views import View
+from game.igdb_wrapper import IGDBRequestsHandler
+from game.twitter_wrapper import TwitterWrapper
+from django.core.paginator import Paginator
 
 
-class MainPageView(TemplateView):
-    template_name = 'game/main_page.html'
+class MainPageView(View):
```

Comment:
This can be still inherited from TemplateView, but with overriden `get_context_data()`, see example here: https://docs.djangoproject.com/en/3.2/ref/class-based-views/base/#templateview